src/runtime-rs/config/configuration-dragonball.toml.in

```diff
@@ -97,6 +97,11 @@ default_maxvcpus = @DEFMAXVCPUS_DB@
 # > 5                --> will be set to 5
 default_bridges = @DEFBRIDGES@
 
+# PCI hotplug
+# Enable or disable pci_hotplug
+# will be set to @DEFPCIHOTPLUG@
+pci_hotplug = @DEFPCIHOTPLUG@
```

Comment:
could you add a comment here that this currently is a dragonball-only config, so that other people could know that this config won't take effect on other VMMs?